client/components/accordion/docs/example.jsx

```diff
@@ -11,7 +11,7 @@ var Accordion = require( 'components/accordion' ),
 	Gridicon = require( 'components/gridicon' );
 
 module.exports = React.createClass( {
-	displayName: 'Accordions',
+	displayName: 'Accordion',
```

Comment:
I think I mentioned that we probably want the displayNames to be `AccordionExample` and then filter out the Example from the titles, etc.